src/core/app/authHooks.ts

```diff
@@ -0,0 +1,96 @@
+import { appConfig } from '~core/app_config';
+import { currentUserAtom, eventFeedsAtom, featureFlagsAtom } from '~core/shared_state';
+import { yandexMetrics } from '~core/metrics';
+import { loadFeatures, setFeatures } from './features';
+import type { AuthSuccessResponse } from '~core/auth/client/AuthClient';
+import type { CurrentUser } from './user';
+import type { ApiClient } from '~core/api_client';
+import type { BackendFeed } from '~core/auth/types';
+
+export async function onPublicLogin(apiClient: ApiClient) {
+  // load features for public user, without token
+  const featuresResponse = await loadFeatures(apiClient, false);
+  setFeatures(featuresResponse);
+}
```

Comment:
why not name it smth like loadPublicFeatures? If that's what it does

Reply:
yeah, in that case we can remove this comment on line 11